### PR TITLE
fail -> raise

### PR DESCRIPTION
**Pull request by ebrohman**
https://github.com/bbatsov/ruby-style-guide/issues/512

### CODE REVIEW

**Comment by adsteel** [comment]
I support this. The Ruby style guide now supports it as well. https://github.com/bbatsov/ruby-style-guide/pull/517

Not sure who should approve these PRs, but it gets my 👍

File: .rubocop.kapost.yml [hunk 1]
```diff
@@ -99,3 +99,6 @@ Lint/PercentStringArray:
   Exclude:
     # SecureHeaders needs the single quotes in `%w[https: 'self']`
     - config/initializers/secure_headers.rb
+Style/SignalException:
+  Description: 'Checks for proper usage of fail and raise.' # the proper usage is 'raise'
```
Comment (ianhenrysmith) [inline .rubocop.kapost.yml:103]: nitpick: use double quotes instead of single

**Review by ianhenrysmith — COMMENTED** [review]
I like this change. A semantic distinction between `raise` and `fail` isn't helpful.

**Comment by derekclee** [comment]
I like this change too. I always had to remember to not use `raise` and I didn't know exactly why other than rubocop complaining. 👍

**Pull request merged**